packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -100,10 +101,8 @@ export function getBeaconPoolApi({
         })
       );
 
-      if (errors.length > 1) {
-        throw Error("Multiple errors on submitPoolAttestations\n" + errors.map((e) => e.message).join("\n"));
-      } else if (errors.length === 1) {
-        throw errors[0];
+      if (errors.length > 0) {
+        throw new MultipleError("Some errors on submitPoolAttestations", errors);
```

Comment:
The previous error was not really good, we don't need to include the operationId (`submitPoolAttestations`) in the error message as the caller knows what method was called and the error log already includes it

https://github.com/ChainSafe/lodestar/blob/06210efab54433a65f275de367c6f8e17dbd4a62/packages/beacon-node/src/api/rest/base.ts#L114


What about we use the error.message as message if `errors.length ===1`, and otherwise log somehting like `Multiple errors submitting attestations` (or similar)

Reply:
Yeah that makes sense.

Reply:
> What about we use the errors[0].message as message if errors.length ===1, and otherwise log something like Multiple errors submitting attestations (or similar)

Actually I'm not sure of this anymore.

Seeing the same error message repeated(for length === 1) feels weird. I feel having a simple `Some errors submitting attestations` regardless of length is better. The actual error message is in the `failures` array regardless.

```
{
    "code": 400,
    "message": "ATTESTATION_ERROR_BAD_TARGET_EPOCH",
    "failures": [
        {
            "index": 0,
            "message": "ATTESTATION_ERROR_BAD_TARGET_EPOCH"
        }
    ]
}
```

vs

```
{
    "code": 400,
    "message": "Some errors submitting attestations",
    "failures": [
        {
            "index": 0,
            "message": "ATTESTATION_ERROR_BAD_TARGET_EPOCH"
        }
    ]
}
```

Reply:
ah right that's a good point, we use [`AttestationErrorCode`](https://github.com/ChainSafe/lodestar/blob/06210efab54433a65f275de367c6f8e17dbd4a62/packages/beacon-node/src/chain/errors/attestationError.ts#L5) as message, that's definitely not ideal, your suggestion make sense. 

But I noticed we need to be careful that we don't lose information here because on the validator client side (http client) we only use the message of the error. We need to make sure to properly handle indexed errors
https://github.com/ChainSafe/lodestar/blob/06210efab54433a65f275de367c6f8e17dbd4a62/packages/api/src/utils/client/httpClient.ts#L376-L379